### ISSUE TITLE
Empty table header in Search Results table

### ISSUE THREAD

**Issue by JessicaMarine** [ustaxcourt/ef-cms#7816]
The first table header (`<th>`) in Search Results table contains no text.

#### Why It Matters
The `<th>` element helps associate table cells with the correct row/column headers. A `<th>` that contains no text may result in cells with missing or incorrect header information.

#### How to Fix It
If the table cell is a header, provide text within the cell that describes the column or row. If the cell is not a header or must remain empty (such as the top-left cell in a data table), make the cell a `<td>` rather than a `<th>`.

#### Steps to Reproduce 
- Navigate to homepage
- Click Order tab
- search orders by keyword "order"
- View search results

**Comment by adunkman** [comment]
Since order search results cannot be currently viewed, I verified this fix via code review and ensured the search results table for cases is read well via screenreader. 

Looks good! 👍
